implementation/src/main/java/io/smallrye/mutiny/operators/multi/overflow/MultiOnOverflowBufferOp.java

```diff
@@ -70,17 +72,42 @@ public void onSubscribe(Subscription subscription) {
         @Override
         public void onItem(T t) {
             if (!queue.offer(t)) {
-                BackPressureFailure ex = new BackPressureFailure(
-                        "Buffer is full due to lack of downstream consumption");
+                BackPressureFailure bpf = new BackPressureFailure("Buffer is full due to lack of downstream consumption");
```

Comment:
I got a few questions about this message. 
Should we make it clearer, like:

"The stream is using a buffer to handle overflows, i.e. the upstream sending more items than the downstream capacity. However, the buffer is reached its maximum capacity. This is due because the upstream sent too many items or because the downstream didn't emit enough requests. "

Reply:
Yes, something shorter, but more explicit since that current message is a bit vague